koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -450,7 +451,7 @@ def convert_to_parquet(self):  # noqa: C901
                 failed_conversion.append(csv_filename)
                 continue
             if self.provider_type == Provider.PROVIDER_OCI:
-                file_specific_start_date = csv_filename.split(".")[1]
+                file_specific_start_date = str(csv_filename).split(".")[1]
```

Comment:
Would `csv_filename.stem` work here?

Reply:
Truthfully? I don't know. This filename comes from here:
https://github.com/project-koku/koku/blob/main/koku/masu/external/downloader/oci/oci_report_downloader.py#L65
`.stem` would probably work here, but I'd rather just leave this one alone since I'm not trying to touch anything OCI related.

Reply:
Perfectly reasonable. I mainly bring it up because `pathlib` has a lot of interesting ways to slice and dice paths elegantly. I haven't mastered it yet, but enjoy discussing different ways to solve things.